### PR TITLE
Fix DockerClientConfig serialization.

### PR DESCRIPTION
**Pull request by KostyaSha**
Resolves #376

### CODE REVIEW

**Comment by KostyaSha** [comment]
:-1: forgot extract serialver

File: src/test/java/com/github/dockerjava/core/RemoteApiVersionTest.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import org.testng.annotations.Test;
+
+/**
+ * @author Kanstantsin Shautsou
+ */
+public class RemoteApiVersionTest {
+
+    @Test
+    public void testSerial() {
+        SerializationUtils.serialize(RemoteApiVersion.unknown());
```
Comment (lanwen) [inline src/test/java/com/github/dockerjava/core/RemoteApiVersionTest.java]: you should serialize some value and then deserialize it to check is it has equal field values

File: src/main/java/com/github/dockerjava/core/RemoteApiVersion.java [hunk 2]
```diff
@@ -15,7 +16,7 @@
  *
  * @author Marcus Thiesen
  */
-public class RemoteApiVersion {
+public class RemoteApiVersion implements Serializable {
```
Comment (lanwen) [inline src/main/java/com/github/dockerjava/core/RemoteApiVersion.java]: what about serialVersionId?

Comment (KostyaSha) [inline src/main/java/com/github/dockerjava/core/RemoteApiVersion.java]: Test ate my head, will do of course https://github.com/docker-java/docker-java/pull/377#issuecomment-158441711

Comment (KostyaSha) [inline src/main/java/com/github/dockerjava/core/RemoteApiVersion.java]: Done.

**Comment by KostyaSha** [comment]
@marcuslinke @lanwen please review, `equals` should be enough imho.

**Comment by lanwen** [comment]
looks fine :+1:

**Pull request merged**